src/mono/wasm/runtime/types/index.ts

```diff
@@ -83,8 +86,73 @@ export type MonoConfig = {
      * application environment
      */
     applicationEnvironment?: string,
+
+    /**
+     * Gets the application culture. This is a name specified in the BCP 47 format. See https://tools.ietf.org/html/bcp47
+     */
+    applicationCulture?: string,
+
+    /**
+     * Overrides the built-in boot resource loading mechanism so that boot resources can be fetched
+     * from a custom source, such as an external CDN.
+     * @param type The type of the resource to be loaded.
+     * @param name The name of the resource to be loaded.
+     * @param defaultUri The URI from which the framework would fetch the resource by default. The URI may be relative or absolute.
+     * @param integrity The integrity string representing the expected content in the response.
+     * @returns A URI string or a Response promise to override the loading process, or null/undefined to allow the default loading behavior.
+     */
+    loadBootResource?: LoadBootResourceCallback;
```

Comment:
Consider that `MonoConfig` is data not functions (mostly). Perhaps we should add separate API for `loadBootResource` and `libraryInitializers`